Harubee-iOS/Sources/Presentation/Components/FloatingTitleTextField.swift

```diff
@@ -12,7 +12,9 @@ import DesignSystem
 struct FloatingTitleTextField: View {
   var title: String
   @Binding var text: String
+  @State var shouldShowKeyboard: Bool
   @State private var isFocused: Bool = false
```

Comment:
private 접근지정자로 변경해주세요
그리고 shouldShowKeyboard를 디폴트 값으로 false를 주는 것이 어떨까요?
키보드가 바로 올라와야 하는 경우가 메모 입력뷰에서만 일어나고 있는 거 같아서 제안드립니다.

```suggestion
struct FloatingTitleTextField: View {
  @State var shouldShowKeyboard: Bool = false
```

Reply:
```
@State private var shouldShowKeyboard: Bool = false
```
이렇게 수정했을 때 메모 입력 뷰에서 `true` 값을 전달해주지 못하는 것 같아요 ..!! ,,

Reply:
```suggestion
struct FloatingTitleTextField: View {
  private var title: String
  @Binding private var text: String
  @State private var shouldShowKeyboard: Bool
  @State private var isFocused: Bool = false
  @FocusState private var isTextfieldFocused: Bool
  
  init(title: String, text: Binding<String>, shouldShowKeyboard: Bool = false) {
    self.title = title
    self._text = text
    self.shouldShowKeyboard = shouldShowKeyboard
  }
```

이렇게 하면 default값으로 줄 수 있어용
```swift
// shouldShowKeyboard가 false이고 싶을때
FloatingTitleTextField(title: "이름", text: $fixedExpenseName)

// shouldShowKeyboard가 true이고 싶을때
FloatingTitleTextField(title: "이름", text: $fixedExpenseName, shouldShowKeyboard: true)

Reply:
옹 해결했어용 감사합니다